server/services/messaging/message-receiver.js

```diff
@@ -1,34 +1,26 @@
 const { ReceiveMode } = require('@azure/service-bus')
-const { getReceiverConfig } = require('./config-helper')
 const MessageBase = require('./message-base')
 
 class MessageReceiver extends MessageBase {
-  constructor (name, config) {
+  constructor (name, config, credentials, action) {
     super(name, config)
```

Comment:
You need to pass through the credentials to `super`.

Reply:
Thanks. Changes made.